bundles/org.openhab.binding.miio/src/main/java/org/openhab/binding/miio/internal/robot/RRMapDraw.java

```diff
@@ -302,32 +308,35 @@ private void drawGoTo(Graphics2D g2d, float scale) {
-            textPos = (int) (ohLogo.getWidth() * scale / 2 + offset);
-            AffineTransform at = new AffineTransform();
-            at.scale(scale / 2, scale / 2);
-            AffineTransformOp scaleOp = new AffineTransformOp(at, AffineTransformOp.TYPE_BILINEAR);
-            g2d.drawImage(ohLogo, scaleOp, offset, height - (int) (ohLogo.getHeight() * scale / 2) - offset);
+            if (image != null) {
+                BufferedImage ohLogo = ImageIO.read(image);
+                textPos = (int) (ohLogo.getWidth() * scale / 2 + offset);
+                AffineTransform at = new AffineTransform();
+                at.scale(scale / 2, scale / 2);
+                AffineTransformOp scaleOp = new AffineTransformOp(at, AffineTransformOp.TYPE_BILINEAR);
+                g2d.drawImage(ohLogo, scaleOp, offset, height - (int) (ohLogo.getHeight() * scale / 2) - offset);
+            } else {
+                logger.debug("Error loading image ohlogo.png: File not be found.");
+            }
         } catch (IOException e) {
-            // no joy
+            logger.debug("Error loading image ohlogo.png:: {}", e.getMessage());
         }
         Font font = new Font("TimesRoman", Font.BOLD, 14);
```

Comment:
You are still using TimeRoman here.

Reply:
yes, I think something went wrong with the final save. Should be fixed now